tracee-ebpf/tracee/argprinters.go

```diff
@@ -10,7 +11,7 @@ import (
 func (t *Tracee) parseArgs(event *external.Event) error {
 	for i := range event.Args {
 		if ptr, isUintptr := event.Args[i].Value.(uintptr); isUintptr {
-			event.Args[i].Value = fmt.Sprintf("0x%X", ptr)
+			event.Args[i].Value = "0x" + strconv.FormatUint(uint64(ptr), 16)
```

Comment:
might be a silly question, but are we sure this won't case a regression? meaning is `strconv.FormatUint(uint64(ptr), 16)` outputting the same as `%X`?

Reply:
Not silly at all, as there is a difference - digits won't be capitalized as before.
Anyway, I took this into consideration before I merged it. Argprinters are used for pretty printing, and eventually we will want to get rid of this extra parsting step and send "raw" events to tracee-rules. For now, even though we still have the argprinters set, no signature expects pointers, so that shouldn't cause any regressions